### PR TITLE
Changes/17

### PR DESCRIPTION
**Pull request by cmadland**
Pull Request for changes to Nov 22.

### CODE REVIEW

**Comment by cmadland** [comment]
Added Naomi to the request

File: Assignments/apa-formatting-requirements.md [hunk 1]
```diff
@@ -15,15 +15,15 @@
...
-| In-text citations | APA style in-text citations used throughout document.  | Some APA style in-text citations used in the document.  | Limited or no APA style in-text citations used throughout document. | No APA style in-text citations used throughout document.  |
+| Title page | Title Page contains all parts and correct format. \(i.e., header and page numbering at top right, running head, title of paper, student's name, college name double-spaced, etc.\) | Title page included with all necessary components, with exception of running head. \(i.e., header and page numbering at top right, running head, title of paper, student's name, college name double-spaced, etc.\) | Title page included, but is missing information \(i.e., header and page numbering at top right, running head, title of paper, student's name, college name double-spaced, etc.\) | None present. |
+|  | Reference List | Reference page follows APA guidelines outlined in  2.11; 6.22; 6.27-6.30. | Reference page mostly follows APA guidelines outlined in 2.11; 6.22; 6.27-6.30.e.,Reference page contains many APA errors. | Reference page lacks use of APA guidelines for references. |
```
[the remark this answers is not in the document]
Comment (NaomiEdit) [reply Assignments/apa-formatting-requirements.md]: I see the Reference row appears higher up on the chart than it did originally. I certainly didn't intend that to happen or for information to be lost. My apologies and glad you caught that. I'm not sure what happened there.

[the remark this answers is not in the document]
Comment (cmadland) [reply Assignments/apa-formatting-requirements.md]: I think this is just a Markdown error. The only differences in the characters between the two versions are the added '|' and 'e.,' on the right, and the removed '|' on the left.
screen shot 2017-11-24 at 9 43 46 am

The vertical bars are vertical gridlines in the table, so moving them can have weird consequences.

[the remark this answers is not in the document]
Comment (cmadland) [reply Assignments/apa-formatting-requirements.md]: Ok...all fixed.

for reference...
screen shot 2017-11-24 at 9 51 57 am

File: Unit-5/Overview.md [hunk 2]
```diff
@@ -1,26 +1,28 @@
...
+In unit 4, as we explored the introduction to research reports, we saw that the research problem establishes why a particular research study is worth doing, the review of literature establishes what is already known about the problem, and the research purpose establishes what the research specifically intends to do.  In unit 5 we want to extend this to the next major section of a research report: The methods and results. In this unit, we will see that the methods section indicates the overall plan for how the research was conducted, how the data was collected, and what the analysis of the data tells us. hese may be termed research design, method, and analysis and results, respectively.  Because of the great differences in quantitative and qualitative methods, we will break this discussion into two units.  This unit will focus on the methods and results of quantitative research reports, while unit 6 will focus on those of qualitative research reports.
```
Comment (HeatherStrong11) [inline Unit-5/Overview.md]: It looks like there is a "T" missing from 4th sentence in this section.  Did that get deleted by accident?

Comment (HeatherStrong11) [reply Unit-5/Overview.md]: From the beginning of the sentence from the word "These"

File: Resources.md [hunk 3]
```diff
@@ -1,8 +0,0 @@
----
```
[the remark this answers is not in the document]
Comment (cmadland) [reply Resources.md]: Ahh...that was my addition. Sorry for not mentioning that.

It is a header code known as YAML (Yet Another Markup Language), and I'm testing it for an automated sync between WordPress and GitHub. If you'd like to know more, let me know.

You can leave it as is.

File: Unit-5/Overview.md (hunk 2 above)
Comment (cmadland) [reply Unit-5/Overview.md]: Sorted.

**Comment by cmadland** [comment]
Unit 1-5; assignments, overview edits approved by @HeatherStrong11

**Pull request merged**